Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -59,6 +61,7 @@ define([
      * @param {Object} options Object with the following properties:
      * @param {String} options.url The URL of the ArcGIS MapServer service.
      * @param {String} [options.token] The ArcGIS token used to authenticate with the ArcGIS MapServer service.
+     * @param {Promise.<String>} [options.requestNewToken] A callback to retrieve new tokens if its detected that the current token has expired.
```

Comment:
`requestNewToken` is a function, not a `Promise<String>`.  Here's how you specify that a parameter is a callback function with jsdoc: http://usejsdoc.org/tags-callback.html

Reply:
Thanks for this.

26801a9ca40e64314f582ac743c567c13c7de5e5